kuwala/core/canvas/src/pages/DataPipelineManagement.js

```diff
@@ -0,0 +1,114 @@
+                    <thead className={'rounded-t-md uppercase'}>
+                    <tr>
+                        <th className={'sticky top-0 px-6 py-3 text-white bg-kuwala-green'}>Name</th>
+                        <th className={'sticky top-0 px-6 py-3 text-white bg-kuwala-green'}>Status</th>
+                        <th className={'sticky top-0 px-6 py-3 text-white bg-kuwala-green'}>Action</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {dataSource.map((e, i)=>{
+                        return (
+                            <tr
+                                key={i} className={'bg-white border-2 text-center'}
+                            >
+                                <td className={'py-6'}>
+                                    <div className={'flex flex-row ml-8 items-center'}>
+                                        <img
+                                            src={e.logo}
+                                            style={{height: 48, width: 48}}
+                                        />
+                                        <label className={'ml-10 text-lg font-medium'}>{e.name}</label>
+                                    </div>
+                                </td>
+                                <td className={'py-6'}>
+                                    <span
+                                        className={'px-4 py-2 rounded-xl bg-red-400 text-white font-semibold'}
+                                    >
+                                        Inactive
+                                    </span>
+                                </td>
+                                <td className={'py-6 space-x-2'}>
+                                    <Link
+                                        to={'/data-source-config'}
+                                        state={{
+                                            index: i,
+                                        }}
+                                        className={'bg-white px-4 py-2 text-white rounded-md border-2 border-kuwala-green hover:bg-kuwala-bg-gray'}
+                                    >
+                                        <span className={'text-kuwala-green font-semibold'}>Configure</span>
+                                    </Link>
+                                    <Link
+                                        to={'/data-source-preview'}
+                                        state={{
+                                            index: i,
+                                        }}
+                                        className={'bg-white text-kuwala-green px-4 py-2 text-white rounded-md border-2 border-kuwala-green hover:bg-kuwala-bg-gray'}
+                                    >
+                                        <span className={'text-kuwala-green font-semibold'}>Preview Data</span>
+                                    </Link>
```

Comment:
The data preview button should only be displayed when the data source has been configured.